style.css

```diff
@@ -116,27 +156,353 @@ button {
   line-height: 52px;
   letter-spacing: 0.05em;
   text-align: center;
+  color: #172b4d;
+  padding-left: 28px;
+  padding-right: 28px;
+  align-self: center;
 }
 
-.second-section hr {
-  width: 25%;
-  height: 4px;
+.boxes {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 3em;
+}
+
+.box {
+  height: 480px;
+  background-color: #d0d9d4;
+  border-radius: 12px;
+  display: flex;
+  flex-direction: column;
+  align-self: center;
+  justify-content: space-between;
+  margin: 0 auto;
+  min-width: 350px;
+}
+
+.box-menu {
+  background-color: #fff;
+  height: 230px;
+  justify-content: center;
+  border-bottom-left-radius: 12px;
+  border-bottom-right-radius: 12px;
+  display: flex;
+  flex-direction: column;
+}
+
+.box-menu h2 {
+  width: 295px;
+  height: 57px;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  align-self: center;
+}
+
+.box-list {
+  display: flex;
+  flex-direction: row;
+  margin: 27px;
+  gap: 5px;
   text-align: center;
-  margin-left: 38%;
-  background-color: #36b37e;
-  margin-top: 8%;
+  justify-content: center;
+  align-self: center;
+}
+
+.box-list-item {
+  background-color: #d0d9d4;
+  padding: 7px;
+  font-size: 12px;
+  border-radius: 8px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: left;
+}
+
+.box-button {
+  color: #fff;
+  background-color: #36b37f;
+  width: 122px;
+  height: 48px;
+  margin-top: 7px;
+  border-radius: 5px;
+  align-self: center;
 }
 
-@media (max-width: 768px) {
+.box-button:hover {
+  background-color: #36b37f;
+  box-shadow: 1px 2px #d0d9d4;
+}
+
+.box-button:active {
+  background-color: #008552;
+}
+
+#about {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.frameleft {
+  position: absolute;
+  width: 100px;
+  height: 150px;
+  left: 0;
+  top: 420px;
+  overflow: hidden;
+  display: inline-block;
+  overflow-x: hidden;
+}
+
+.frameright {
+  position: absolute;
+  width: 130px;
+  height: 100px;
+  right: -25px;
+  top: -40px;
+  overflow: hidden;
+  display: inline-block;
+  overflow-x: hidden;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.about-desc {
+  width: 45%;
+  font-weight: 600;
+  font-size: 23px;
+  line-height: 32px;
+  text-align: center;
+  color: #42526e;
+  padding-top: 25px;
+  align-self: center;
+}
+
+#about h2,
+#works h3 {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: center;
+}
+
+#about .about-btn {
+  margin: 0;
+  margin-bottom: 30px;
+  margin-top: 30px;
+}
+
+.about-box {
+  border-radius: 12px;
+  display: flex;
+  flex-direction: column;
+  align-self: center;
+  justify-content: space-between;
+  margin: 0 auto;
+  padding: 10px;
+  max-width: 360px;
+  background: #ebf0ee;
+}
```

Comment:
- [OPTIONAL ] Please make sure that your `about me` section looks exactly like the Figma design.
### YOUR DESIGN
![Screenshot from 2023-01-20 00-03-46](https://user-images.githubusercontent.com/92303891/213589250-de98a6aa-fbeb-4a85-a98a-3cb0a566b207.png)

### FIGMA DESIGN
![Screenshot from 2023-01-20 00-03-07](https://user-images.githubusercontent.com/92303891/213589271-22e1fd07-c194-4eb6-95cd-c0c1128df305.png)

- [ OPTIONAL ] Please kindly make sure your project is responsive on Desktop screens. But your project has a horizontal scroll on desktop screens